src/vs/workbench/browser/parts/menubar/menubarPart.ts

```diff
@@ -378,27 +372,34 @@ export class MenubarPart extends Part {
 				titleElement: titleElement
 			});
 
-			// Update cached actions array for CustomMenus
-			const updateActions = () => {
-				this.customMenus[menuIndex].actions = [];
+			const updateActions = (menu: IMenu, target: IAction[]) => {
+				target.splice(0);
```

Comment:
@sbatten what is the purpose of this call?

Reply:
@bpasero we need to update the existing target action array so we need to clear it first.